packages/manager/src/components/RegionSelect/RegionSelect.tsx

```diff
@@ -172,21 +175,24 @@ export const RegionSelect = React.memo((props: RegionSelectProps) => {
         placeholder="Select a Region"
         value={selectedRegion}
       />
-      {showEdgeIconHelperText && ( // @TODO Gecko Beta: Add docs link
-        <StyledEdgeBox>
-          <EdgeRegion />
+      {showDistributedRegionIconHelperText && ( // @TODO Gecko Beta: Add docs link
+        <StyledDistributedRegionBox>
+          <DistributedRegion />
           <Typography
-            data-testid="region-select-edge-text"
+            data-testid="region-select-distributed-region-text"
             sx={{ alignSelf: 'center', textWrap: 'nowrap' }}
           >
             {' '}
-            Indicates an edge region.{' '}
-            <Link aria-label="Learn more about Akamai edge regions" to="#">
+            Indicates an distributed region.{' '}
```

Comment:
```suggestion
            Indicates a distributed region.{' '}
```

Need to adjust the grammar here and in and the tooltip

![Screenshot 2024-05-13 at 12 58 31 PM](https://github.com/linode/manager/assets/114682940/1a0d8da0-8792-431e-b3c3-f9c4c95e9816)